tests/SideBySide/SideBySide.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
  <PropertyGroup Condition=" '$(Configuration)' != 'Baseline' ">
-    <TargetFrameworks>netcoreapp1.1.1;net462</TargetFrameworks>
```

Comment:
`4.6.1` is the lowest desktop target that `netstandard2.0` supports.  It will use the `net451` library as long as we still build that, but if we ever remove `net451`, `4.6.1` will be the lowest level supported.

Moved to `netcoreapp1.1.2` because it is the most recent shared framework for .NET Core `1.x`.  This will test the `netstandard1.3` package, although any version of `netcoreapp1.x` should work.  I just find it's easiest to bring them up to the latest.